index.md

```diff
@@ -2,8 +2,17 @@
 site: sandpaper::sandpaper_site
 ---
 
-This is a new lesson built with [The Carpentries Workbench][workbench]. 
+This lesson provides an overview of the fundamentals of imaging data that has been acquired through light microscopy experiments. This course is aimed at all researchers who are looking to work with light microscopy imaging data, or to refresh their knowledge on some of the key fundamentals.
 
+::::::::::::::::::::::::::::::::::::::::::  prereq
 
-[workbench]: https://carpentries.github.io/sandpaper-docs
+## Prerequisites
+
+- Some prior experience with acquiring/working with light microscopy data would be useful, but isn't necessary. This course is aimed at anyone who is looking to work with light microscopy imaging data in their research projects.
+
+- No python experience is necessary or assumed, but you will get more out of the course if you know some basics e.g. from the [Software Carpentry python course](https://swcarpentry.github.io/python-novice-inflammation/)
```

Comment:
Do we want a link to UCL SC courses as well in case they want to have a guided version?

Reply:
Sounds like a good plan - do you know what the link would be for this?

Reply:
@dpshelio Is [this page](https://www.ucl.ac.uk/advanced-research-computing/education/training) the best one to use for ARC software carpentry courses? It seems to mention the November course that just took place amongst other offerings.
I looked at [Research Software London](https://rslondon.ac.uk/), but it appears not to have been updated since July :(